xtremedata-dbx-cluster-centos/azuredeploy.json

```diff
@@ -0,0 +1,452 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "String",
+            "defaultValue": "West US",
+            "allowedValues": [
+                "West US"
+            ],
+            "metadata": {
+                "description": "Azure cloud location for dbX cluster"
+            }
+        },
+        "newStorageAccountName": {
+            "type": "String",
+            "defaultValue": "dbxstrg",
+            "metadata": {
+                "description": "Prefix for new storage account that will be created to store virtual machine VHDs"
+            }
+        },
+        "storageAccountType": {
```

Comment:
if there is only one allowed value, consider making this a variable so that it's one less parameter for the user to enter